.yamato/standalone-project-tests.yml

```diff
@@ -0,0 +1,35 @@
+{% metadata_file .yamato/project.metafile %}
+---
+
+# For every platform and editor version, run its project tests without
+# running package tests too since they are handled on their respective
+# jobs
+{% for project in projects -%}
+{% for editor in project.test_editors -%}
+{% for platform in test_platforms -%}
+standalone_test_{{ project.name }}_{{ editor }}_{{ platform.name }}:
```

Comment:
This feels like the wrong approach. You're doing a bunch of work to call into a shell script that ultimately only runs a few commands, which are almost identical across platforms. This obfuscates some of what's going on, it duplicates a bunch of code, and it introduces additional places where we have to make changes anytime we update things like editors or platforms. You also can't benefit from dependency graph stuff in the yamato job scheduler.

Reply:
I think if you, instead, had a build & test job just like you did with mobile you could do this all without the batch files.

The build & test commands are almost identical - the test one could look like this for example (build would be very similar):

```
{% for project in projects -%}
{% for editor in project.test_editors -%}
{% for platform in test_platforms -%}
standalone_test_{{ project.name }}_{{ editor }}_{{ platform.name }}:
  name : standalone {{ project.name }} tests - {{ editor }} on {{ platform.name }}
  agent:
    type: {{ platform.type }}
    image: {{ platform.image }}
    flavor: {{ platform.flavor}}
  commands:
    - {% if platform.name == "win"}SET{% else %}EXPORT{% endif %} UTR_VERSION=0.12.0
    - {% if platform.name != "win" %}./{% endif %}utr --suite=playmode --platform={{ platform.utr_name }}  --editor-location=.Editor --testproject=testproject --player-save-path=build/players --artifacts_path=build/logs --scripting-backend=mono --build-only --testfilter=Unity.Netcode.RuntimeTests --extra-editor-arg=-batchmode --extra-editor-arg=-nographics
{% endfor -%}
{% endfor -%}
{% endfor -%}
```

Reply:
@JesseOlmer are you satisfied with the changes I've made here?